content/en/docs/reference/kubectl/config_management/introduction.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Introduction"
+linkTitle: "Introduction"
+weight: 1
+type: docs
+description: >
+    Introduction to App Management
+---
+
+
+
+{{< alert color="success" title="TL;DR" >}}
+- Apply manages Applications through files defining Kubernetes Resources (i.e. Resource Config)
+- Kustomize is used to author Resource Config
+{{< /alert >}}
```

Comment:
Thanks.
We've realized that a straight conversion isn't right. This proposed tweak should help though:
```suggestion
```